DSCResources/MSFT_xWebsite/MSFT_xWebsite.schema.mof

```diff
@@ -10,17 +10,17 @@ Class MSFT_xWebBindingInformation
 };
 
 
-[ClassVersion("2.0.0"), FriendlyName("xWebsite")] 
+[ClassVersion("2.0.0"), FriendlyName("xWebsite")]
 class MSFT_xWebsite : OMI_BaseResource
 {
   [Key] string Name;
-  [write] string PhysicalPath;
+  [Required] string PhysicalPath;
```

Comment:
I said to do this didn't I?

There are two cases I don't think I considered:

* Updating the default site, you don't want to specify the Physical Path because usually you are basically disabling the site.
* Stopping a site, again you don't want to specify a path, because it no longer matters.